### PR TITLE
solution

### PR DESCRIPTION
**Pull request by kq333**
[DEMO LINK](https://kq333.github.io/react_sum/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
```
Comment (d-wasilewski) [inline src/App.jsx:10]: checklist 1

File: src/components/Sum.jsx [hunk 2]
```diff
@@ -1 +1,22 @@
...
+      {' '}
+      {a}
+      {' '}
+      and
+      {' '}
+      {b}
+      {' '}
+      is
+      {' '}
+      {a + b}
```
Comment (d-wasilewski) [inline src/components/Sum.jsx:17]: checklist 2

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

File: src/components/Sum.jsx [hunk 3]
```diff
@@ -1 +1,11 @@
...
+
+const Sum = ({ a = 0, b = 0 }) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
+);
+
+export default Sum;
```
Comment (d-wasilewski) [inline src/components/Sum.jsx:11]: you dont have to export `default` when you import it with { }, that;s why checks fail

File: src/components/Sum.jsx [hunk 4]
```diff
@@ -1 +1,11 @@
-// export function Sum() {}
+import React from 'react';
+
+const Sum = ({ a = 0, b = 0 }) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```
Comment (d-wasilewski) [inline src/components/Sum.jsx:8]: you don't need this fragment wrapper here

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

File: src/components/Sum.jsx [hunk 5]
```diff
@@ -1 +1,11 @@
...
+
+const Sum = ({ a = 0, b = 0 }) => (
+
+  <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
+
+);
+
+export default Sum;
```
Comment (d-wasilewski) [inline src/components/Sum.jsx:11]:
```suggestion
export Sum;
```

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

File: src/components/Sum.jsx (hunk 5 above)
Comment (d-wasilewski) [inline src/components/Sum.jsx:11]: fix the test

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

**Review by kasiakubisiak — APPROVED** [review]

**Pull request closed without merging**